trade-finance-manager-api/src/v1/api.js

```diff
@@ -340,25 +340,22 @@ const updateFacility = async ({ facilityId, tfmUpdate, auditDetails }) => {
 };
 
 const createFacilityAmendment = async (facilityId, auditDetails) => {
```

Comment:
Can we wrap this in `try-catch` block please, in case the API call fails.

Reply:
We don't want to I think -- as we handle `AxiosError` at the controller level (changing the response), and all the `ApiErrors` at the global middleware level -- Catching it here would still mean we'd have to handle these errors at the controller level, and we'd want to keep the grainularity of the errors, so we'd just rethrow